changelog/lambdacomp.dd

```diff
@@ -0,0 +1,10 @@
+Lambda comparison using __traits(isSame, ...)
+
+It is now possible to compare two lambda functions, under certain
+constraints, using __traits(isSame, lamda1, lambda2). In order to
+correctly compare two lambdas, the following conditions must be
+satisfied:
+
+-> the lambda function arguments must not have a template instantiation as an explicit argument type. Any other argument types (basic, user-defined, template) are supported.
+
+-> the lambda function body must contain a single expression (no return statement) which contains only numeric values, manifest constants, enum values and arguments. If the expression contains local variables, function calls or return statements, the function is considered uncomparable.
```

Comment:
more rational wanted.
why are the limitations the way they are ?

Reply:
That's the point I got to. Later on, the limitations will be lifted (where possible).